src/components/QuestionnaireSection.jsx

```diff
@@ -36,6 +47,19 @@ class QuestionnaireSection extends React.Component {
 		this.getQuestionCard(id).classList.remove('error');
 	}
 
+	handleSelect(e) {
+		if (e.keyCode != 32) {
```

Comment:
Consider using a named constant for space like "KEY_SPACE" instead of hardcoding "32" if 32 isn't well-known.